styles.css

```diff
@@ -0,0 +1,53 @@
+
+body {
+  margin: 0%;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+.hamb {
+  margin-top: 19px;
+  margin-right: 16px;
+}
+
+.mix1 {
+  margin-top: 32.5px;
+}
+
+h1 {
+  padding-top: 30px;
+  font-family: 'Playfair Display', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 50px;
+  color: #1a2236;
+}
+
+.firstp {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+}
+
+#greetings {
+  display: flex;
+  flex-direction: column;
+}
+
+.start {
+  width: 288px;
+  height: 50px;
+  color: white;
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: 500;
+  align-self: center;
+  background: #1a2236;
+}
```

Comment:
Kindly add the corresponding button effects to your buttons:
![image](https://user-images.githubusercontent.com/42799579/198192213-daca1fd0-825f-423d-9c7a-0ed4724d3d4f.png)

You should implement the hover & pressed effects 😃.

Also, remove your button border to look as the one in the design:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/198192383-72c583c4-3478-474a-ada1-36aa1864076a.png)  | ![image](https://user-images.githubusercontent.com/42799579/198192401-7309ef9d-825e-41b1-8bf6-f25d93425245.png)  |

You can do it by simply adding:
```css
button {
  border: 0;
}
```